code/game/objects/items/devices/radio/radio_objects.dm

```diff
@@ -138,6 +138,14 @@ GLOBAL_LIST_EMPTY(deadsay_radio_systems)
 	to_chat(user, "<span class='notice'>You <b>[broadcasting ? "enable" : "disable"]</b> [src]'s hotmic!</span>")
 	add_fingerprint(user)
 
+/obj/item/radio/CtrlShiftClick(mob/user)
+	if(user.stat || HAS_TRAIT(user, TRAIT_HANDS_BLOCKED) || !Adjacent(user) || !istype(user))
```

Comment:
```suggestion
	if(!istype(user) || user.stat || HAS_TRAIT(user, TRAIT_HANDS_BLOCKED) || !Adjacent(user))
```
istype check should come first

Reply:
Also remember that we don't need to check `user.stat`. The hands blocked trait is automatically added to unconscious/dead mobs.